### DISCUSSION TITLE
Xブーリアン→〇論理（値の）/ブール

### DISCUSSION THREAD

**Discussion by koizumistr** [pgsql-jp/jpug-doc#2495] [NGとする語]
#2494 です。

ここではまだ論理値vsブール値の決着はつけなくてもいいと思いますけど、
流石に「ブーリアン値」というのはありえないな、ということで。

**Comment by noborus** [comment]
「Boolean value」が対象

**Comment by noborus** [comment]
「値」ではないブーリアンは良いのでしょうか？
ブーリアン結果、ブーリアン列、ブーリアン演算子がありますね。

**Reply by koizumistr** [reply]
個人的にはすべて却下ですけど、皆さんはいかがでしょう。

「好きではない」度は↓な感じ。
ブーリアン結果 xxxxx
ブーリアン列 xxx
ブーリアン演算子 xx

ただし、「ブーリアン」単独だったら見逃してやってもいいかな、みたいな。（あ、でも形容詞として使っているはずだから全滅か？）

**Reply by noborus** [reply]
今あるのを見てみましたが、単純にブーリアンをすべて置き換えてNGにして良さそうです。

**Reply by koizumistr** [reply]
> ブーリアン結果、ブーリアン列、ブーリアン演算子がありますね。

三種類もあるなんて、、、と思ったですが、それぞれ一つずつなんですね。であれば、
> 単純にブーリアンをすべて置き換えてNGに

もありと思います。「ブーリアン」がいいと言う人がいなければ、対処してしまおうと思いますが、どなたか「『ブーリアン』がいいと言う人」いらっしゃいますか。

#2494 は一旦保留ということで。

**Reply by koizumistr** [reply]
特に反対意見もないようなので、「ブーリアン」をNGにするよう #2494 を変更しました。
